main.go

```diff
@@ -49,6 +50,8 @@ func addFlags(s *cmd.Server, fs *pflag.FlagSet) {
 	fs.BoolVar(&s.AddIPTablesRule, "iptables", false, "Add iptables rule (also requires --host-ip)")
 	fs.StringVar(&s.HostInterface, "host-interface", "docker0", "Host interface for proxying AWS metadata")
 	fs.StringVar(&s.HostIP, "host-ip", s.HostIP, "IP address of host")
+	fs.DurationVar(&s.BackoffMaxInterval, "backoff-max-interval", backoff.DefaultMaxInterval, "Max interval for backoff when querying for role.")
```

Comment:
I think I'd rather change the defaults to some saner values. We probably don't want to retry for more than a few seconds actually, what do you think?